openedx/core/djangoapps/agreements/api.py

```diff
@@ -0,0 +1,55 @@
+"""
+Honor code API
+"""
+
+import logging
+
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
```

Comment:
Instead of using the User object directly, do `get_user_model`. See example at https://github.com/edx/edx-platform/blob/3152c14192dcccac79b7b7a76eba91b380eba78a/openedx/core/djangoapps/user_api/verification_api/views.py#L3